### DISCUSSION TITLE
MonadTall doesn’t grow windows

### DISCUSSION THREAD

**Discussion by DasLixou** [qtile/qtile#3736] [Q&A] [answered]
Using monadtall and keybindings to lazy.layout.grow_left() and so on.. in layout columns it works but in monadtall they don’t have any effect

**Comment by elParaguayo** [comment, marked as answer]
Correct. The XMonad layouts don't have `grow_left/right` they just have `grow()`.

**Reply by DasLixou** [reply]
Is there a way to perform multiple functions on one keybinding?

**Reply by elParaguayo** [reply]
Yes, you can add multiple commands to the same key:
``` python
Key([mod, "shift"], "h", lazy.layout.grow_left(), lazy.layout.grow())
```

**Reply by DasLixou** [reply]
thanks :D

**Reply by ervinpopescu** [reply]
> Yes, you can add multiple commands to the same key:
> ``` python
> Key([mod, "shift"], "h", lazy.layout.grow_left(), lazy.layout.grow())
> ```

Doesn't this pollute the logs? :)

**Reply by elParaguayo** [reply]
That's a fair question. You could do this instead:
``` python
Key(
    [mod, "shift"], "h",
    lazy.layout.grow_left().when(layout=["bsp", "columns"]),
    lazy.layout.grow().when(layout=["monadtall", "monadwide", "monadthreecol"])
)
```
